_data/versions.yml

```
@@ -16,17 +16,17 @@
 # Database of the current version
 #
 current:
-  number: '13.0.0'
-  pinned_number: '13.0.*'
-  major_number: '13'
-  date: 2023-08-23
-  git-tag: 'b7d2f7ffca66c868bd2fce5b3749c6caa002a7f0'
-  github-tag-link: 'https://github.com/apache/arrow/releases/tag/apache-arrow-13.0.0'
-  release-notes: 'https://arrow.apache.org/release/13.0.0.html'
-  mirrors: 'https://www.apache.org/dyn/closer.lua/arrow/arrow-13.0.0/'
-  tarball-name: 'apache-arrow-13.0.0.tar.gz'
-  tarball-url: 'https://www.apache.org/dyn/closer.lua?action=download&filename=arrow/arrow-13.0.0/apache-arrow-13.0.0.tar.gz'
-  java-artifacts: 'https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.arrow%22%20AND%20v%3A%2213.0.0%22'
-  asc: 'https://downloads.apache.org/arrow/arrow-13.0.0/apache-arrow-13.0.0.tar.gz.asc'
-  sha256: 'https://downloads.apache.org/arrow/arrow-13.0.0/apache-arrow-13.0.0.tar.gz.sha256'
-  sha512: 'https://downloads.apache.org/arrow/arrow-13.0.0/apache-arrow-13.0.0.tar.gz.sha512'
+  number: '14.0.0'
+  pinned_number: '14.0.*'
+  major_number: '14'
+  date: '2023-11-01'
```

Oh, I forgot to remove quotation in apache/arrow#37985 ...

```suggestion
  date: 2023-11-01
```